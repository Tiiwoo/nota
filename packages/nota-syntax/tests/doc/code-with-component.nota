%let x = @em{**content**}
#x