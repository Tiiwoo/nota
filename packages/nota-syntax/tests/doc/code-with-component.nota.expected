[
  ...(() => {
    let x = el(
      "em",
      {
        block: true,
      },
      el("strong", {}, "content")
    );
    return [null, el("p", {}, x)];
  })(),
];